### PR TITLE
Allow users to call a ResourcesRoute from their app.

### PR DESCRIPTION
**Pull request by SeyZ**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request
- [x] Write changes made in the CHANGELOG.md
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by SeyZ** [comment]
[Oyst want to have a Lumber generated admin with multi databases support](https://trello.com/c/0bb74tZm/1727-oyst-want-to-have-a-lumber-generated-admin-with-multi-databases-support)

**Comment by SeyZ** [comment]
With that feature, I can do this easily:

```
var express = require('express');
var router = express.Router();
var liana = require('forest-express-sequelize');
var Serializer = require('forest-express-sequelize').ResourceSerializer;

function orders(req, res, next) {
  req.query.filterType = 'and';
  req.query.filter = { merchant_id: req.params.merchantId };

  liana.ResourcesRoute.order.list(req, res, next);
}

router.get('/merchant/:merchantId/relationships/orders', orders);

module.exports = router;
```

File: index.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ function requireAllModels(Implementation, modelsDir, displayMessage) {
 
 exports.Schemas = Schemas;
 exports.logger = logger;
+exports.ResourcesRoute = {};
```
Comment (arnaudbesnier) [inline index.js]: `ResourcesRoutes` with a `s` seems more understandable.

Comment (arnaudbesnier) [reply index.js]: `RoutesPerModel` is a better name?
`RoutesPerModel.order.list`?

Comment (SeyZ) [reply index.js]: I agree about RoutesPerModel is more explicit. But I also want to be consistent with the other classes: ResourceSerializer, ResourceDeserializer.

If we change, let's schedule a global name migration with backward compatibility. Honestly, I think we can move forward like this. Tell me what you think.

**Pull request merged**